homeassistant/components/sensor/habitica.py

```diff
@@ -0,0 +1,151 @@
+"""
+The Habitica sensor.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.habitica/
+"""
+
+import logging
+from collections import namedtuple
+# from typing import Any
+from datetime import timedelta
+import voluptuous as vol
```

Comment:
Please add a blank line between standard library and 3rd party imports and 3rd party imports and homeassistant imports.